src/gui/MixerView.cpp

```diff
@@ -185,11 +272,61 @@ int MixerView::addNewChannel()
 
 	updateMixerLine(newChannelIndex);
 
+	updateAutoTrackSortOrder();
 	updateMaxChannelSelector();
 
 	return newChannelIndex;
 }
 
+void MixerView::updateAutoTrackSortOrder()
+{
+
+	Mixer * mix = Engine::mixer();
+	QList<int> *list = new QList<int>();
```

Comment:
Should this really be allocated on the heap? A simple ``std::vector`` on the stack would work fine here, no?